mesonbuild/build.py

```diff
@@ -2585,6 +2585,7 @@ def __init__(self,
                  install_tag: T.Optional[T.List[T.Optional[str]]] = None,
                  absolute_paths: bool = False,
                  backend: T.Optional['Backend'] = None,
+                 description: str = 'Generating {} with a custom command'
```

Comment:
trailing comma please :) this isn't JSON, we are actually allowed to have one and it makes it soooo much easier to add additional fields as you have seen here right now. ;)